src/main/java/io/cdap/plugin/gcp/firestore/util/FirestoreUtil.java

```diff
@@ -40,29 +41,82 @@ public class FirestoreUtil {
    *
    * @param serviceAccountFilePath path to credentials defined in JSON file
    * @param projectId Google Cloud project ID
+   * @param databaseId Cloud Firestore Database Id
    * @return Firestore service
    */
-  public static Firestore getFirestore(@Nullable String serviceAccountFilePath, String projectId) {
+  public static Firestore getFirestore(@Nullable String serviceAccount,
+                                       @Nullable Boolean isServiceAccountFilePath,
+                                       String projectId,
+                                       String databaseId) {
     try {
       FirestoreOptions.Builder optionsBuilder = FirestoreOptions.newBuilder()
-        .setProjectId(projectId);
+        .setProjectId(projectId)
+        .setDatabaseId(databaseId);
 
-      LOG.debug("serviceAccount={}, project={}...", serviceAccountFilePath, projectId);
+        LOG.debug("serviceAccount={}, isServiceAccountFilePath={}, project={}, databaseId={}...", serviceAccount, isServiceAccountFilePath, projectId, databaseId);
```

Comment:
Similarly here, please don't log `serviceAccount` even though it was logged before remove it.